www/assessment/reset-response.php

```diff
@@ -0,0 +1,209 @@
+                onclick=resetResponse(1)
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Demo 2  -->
+    <div class="section pad-sml">
+        <h3">Choice Matrix</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_2_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                onclick=resetResponse(2)
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Demo 3  -->
+    <div class="section pad-sml">
+        <h3">Cloze Dropdown</h3>
+        <hr>
+        <span class="learnosity-item" data-reference="demo_reset_response_act_3_no_edit"></span>
+        <div class="button-wrapper">
+            <button
+                type="button"
+                class="resetResponseButton"
+                onclick=resetResponse(3)
+            >
+                <span>Reset Response</span>
+            </button>
+        </div>
+        <div class="button-note">
+            <br>
+            <p>NOTE: The Reset Response button is a custom button that calls the <a href="https://reference.learnosity.com/questions-api/methods#questionsApp-ResetResponse"><code>resetResponse</code></a> public method.</p>
+        </div>
+    </div>
+
+    <!--  Load items API  -->
+    <script src="<?php echo $url_items; ?>"></script>
+    <script>
+        /* Demo 1  */
+        const initializationObjectDemo1 = <?php echo $signedRequestDemo1; ?>;
+        const itemsAppDemo1 = LearnosityItems.init(initializationObjectDemo1);
+
+        /* Demo 2  */
+        const initializationObjectDemo2 = <?php echo $signedRequestDemo2; ?>;
+        const itemsAppDemo2 = LearnosityItems.init(initializationObjectDemo2);
+
+        /* Demo 3  */
+        const initializationObjectDemo3 = <?php echo $signedRequestDemo3; ?>;
+        const itemsAppDemo3 = LearnosityItems.init(initializationObjectDemo3);
+
+        /* Shared function to reset the specified demo question response */
+        function resetResponse(demoNumber) {
+            let question;
+
+            switch (demoNumber) {
+                case 1:
+                    question = itemsAppDemo1.questions()[Object.keys(itemsAppDemo1.questions())[0]];
```

Comment:
This is not a good way to reset, as you are reseting only 1 question. Why don't we loop and reset all questions in that particular item (or all items)

Also, instead of using `switch` let's create a method to reset. Check my overall suggestion (in case you still want to initialize 3 Items apps instead of 1)